index.js

```diff
@@ -41,22 +44,25 @@ getBooksPublishedAfter2010([
 // Q3
 // Write a JavaScript function that takes an array of product objects and returns the total price of all products in the category "electronics".
 function getTotalPriceOfElectronics(products) {
-  // Write code here
+  const electronics = products.map((product) => product.price);
+  return electronics;
```

Comment:
this function will return an array of all product prices, what we need is to return the sum of all products with the category of "electronics".